src/lib/greenboot.rs

```diff
@@ -21,9 +21,9 @@ pub fn run_diagnostics(skipped: Vec<String>) -> Result<Vec<String>> {
 
     // Run required checks
     for path in GREENBOOT_INSTALL_PATHS {
-        let greenboot_required_path = format!("{}/check/required.d/", path);
+        let greenboot_required_path = format!("{path}/check/required.d/");
```

Comment:
Asking for my own knowledge, is this the preferred way to write these format lines?

Reply:
I believe this format is used to keep `clippy` happy; see https://github.com/fdo-rs/fido-device-onboard-rs/commit/8fb27caab27c91782452d0b65cb8267cd032912c for another example